docs/guides/integration-options-private-operator.md

```diff
@@ -31,17 +41,19 @@ The participant must sign a contract (see [Account Setup](../getting-started/gs-
 A Private Operator has no visibility into the raw UID2s or UID2 tokens processed by a Public Operator or another Private Operator. Each Private Operator is isolated from all other Operators.
 :::
 
-## Private Operator Workflow
+## Hosting Options for Private Operators
 
-Every Private Operator runs in one of the following:
+If you choose to be a Private Operator, several implementation options are available. You can do any of the following:
 
-- [Nitro](https://aws.amazon.com/ec2/nitro/) Enclave (AWS)
-- Confidential space (GCP)
-- Confidential computing environment (Azure)
+- Use a cloud services setup. UID2 supports hosting UID2 in an <Link href="../ref-info/glossary-uid#gl-enclave">enclave</Link> on the following cloud service providers (medium level of effort to implement):
+  - [Nitro Enclave](https://aws.amazon.com/ec2/nitro/) from AWS
+  - [Confidential Space](https://cloud.google.com/confidential-computing#confidential-space), a confidential computing option from [Google Cloud](https://cloud.google.com/docs/overview/) Platform
+  - [Confidential Container](https://learn.microsoft.com/en-us/azure/confidential-computing/confidential-containers), a confidential computing option from Microsoft Azure
+- Use your own machines to generate and manage UID2s (greater level of effort to implement).
```

Comment:
We should remove this line - there is no way to run private operators locally

Reply:
Done, thx!